doc/jrebel.md

```diff
@@ -28,16 +28,19 @@ You don't need to integrate with your IDE, since we're using sbt to do the servl
 Fortunately, the gitbucket project is already set up to use JRebel.
 You only need to tell jvm where to find the jrebel jar.
 
-To do so, edit your shell resource file (usually `~/.bash_profile` on Mac, and `~/.bashrc` on Linux), and add the following line:
+To do so, edit your shell resource file (usually `~/.bash_profile` on Mac, and `~/.bashrc` on Linux), and add one of the following lines:
 
 ```bash
-export JREBEL=/path/to/jrebel/legacy/jrebel.jar
+export JREBEL=/path/to/jrebel/lib/libjrebel64.dylib   # New agent on macOS
+export JREBEL=/path/to/jrebel/lib/jrebel64.dll        # New agent on Windows
```

Comment:
Windows is not based on bash :nerd_face: 
Simply, "set your environment variable JREBEL=/path/to..."

Reply:
Oops!